doc/index.rst

```diff
@@ -150,6 +151,9 @@ Optional environment variables
 
 Note that some of these currently tied only to a single option, but we still leave them for the future when more options become available (e.g. ament with BUILDER).
 
+* `ABICHECK_MERGE` (default: not set): Used only when `ABICHECK_URL` is set. For travis it can be set to 'auto' to auto-detect pull requests. If set to 'true' the merge parent (see `Terminology section <#terminology>`_) will be checked against.
```

Comment:
> will be checked against.

Against what ;) ?

Reply:
The packages `industrual_ci` was run for.. as all our tests..

Reply:
Sorry I don't know why I commented this https://github.com/ros-industrial/industrial_ci/pull/199#discussion_r146259313. Please ignore.